packages/trader/src/App/Components/Form/fieldset.tsx

```diff
@@ -45,6 +47,7 @@ const Fieldset = ({
                         <span
                             className={classNames('', {
```

Comment:
Do we need empty string in classNames?

Reply:
It is not related to my changes ,It's old code